eras/conway/impl/src/Cardano/Ledger/Conway/PParams.hs

```diff
@@ -727,6 +769,8 @@ instance Crypto c => ToJSON (ConwayPParams Identity (ConwayEra c)) where
   toJSON = object . conwayPParamsPairs
   toEncoding = pairs . mconcat . conwayPParamsPairs
 
+-- TODO: Add protocolVersion to JSON output (of PParams, NOT PParamsUpdate) with
+-- something like: [("protocolVersion", toJSON $ pp ^. ppProtocolVersionL)]
```

Comment:
@lehins I think I want to add a `protolVersion` entry here, but fail to satisfy constraints because `ProtVerAtMost` on `hkdProtocolVersionL` and couldn't make `ppProtocolVersionL` work. Can you help here?